board_rules.md

```diff
@@ -0,0 +1,83 @@
+Don't forget to link the project Repository.
+
+### Sprint backlog
+
+Before sprint planning the issues place into Sprint backlog by PM.
+During the sprint planning process, the issues from the sprint backlog are estimated (labels with estimation are assigned).
+
+**Estimation** is carried out with the participation of all team members online using the **Scrum Poker** technology on the resource [planningpokeronline](https://planningpokeronline.com).
+
+Except estimation labels in progect also provided labels: 
+
+- `Invalid`- this doesn't seem right.
+- `Documentation` - improvements or additions to documentation.
+- `Need more discussion`- further information is requested.
+- `Bug`- something isn't working.
+- `Blocker` - urgent level of priority, it's blocking another task.
+- `Blocked` - this task cannot be done until the task on which it depends is completed. (Inside the blocked issue is placed a link to the blocker).
+
+All types of labels are marked with separate color.
+
+**Important**: 
+
+* the fields milestone, and labels with estimation points should be filled for the issues after Sprint planning;
+* if the PR is approved, then the PR creator can independently resolove conversations. 
+
+### In Progress
+
+When the work on the task starts it moves to in Progress section and stays there until PR will be opened.
+
+**Important**: the field assignee should be strictly added `In Progress` section. 
+
+### In Review
+
+When the task code is under review, it is located in Progress column.
+When the work on the task is done, the PR is opened and waited for the review process, the PR is placed in the in Review section. In the review column there are only open PR.
+Don't forget to link the opened PR with the issue. For doing that, please, fill the "Develoment section" when describing the issue and choose menu item "Link a branch or pull request" -> select the a repository and PR.
+
+### Done
+
+The issues have successfully gone through the review process and merged into main branch can be moved to the Done section.
+
+## Usage
+
+The issue moves from left to right and can't skip any step (board section). The exception is when the team decides that the task cannot be completed or it needs to be canceled (read the description of **Done** section).
+The main flow is:
+ `
+Common backlog -> Sprint backlog -> In progress -> In review -> Done.
+`
+
+## Important:
+
+### Creating a new branch:
+1) all branches should be created from the main branch;
+2) name of a new branch should show the issue type(feature, bug fix), issue number, and branch name and follow this pattern  [type_of_task - #issue_from_backlog - title]. 
+For example: **feature-1-create-project.**
+
+### Merge restrictions to the main branch:
+- minimum two approvals;
+- all conversations should be closed (resolved) after exhaustive discussion.
+
+## Note
+
+The team has daily stand ups at ten o'clock (GTM+3).
```

Comment:
Maybe just provide the link to the wiki page with the team schedule, to have one source of truth, and do not update the info in different places.
Or just remove this section, it's not related to board rules.

Reply:
Yes, you are right! This information isn't related to board rules. The line is deleted.